### PR TITLE
[dmx] Extend action to full chasers

### PR DESCRIPTION
**Pull request by J-N-K**
Requested in the community forum: support full chaser configuration in rule action.

Signed-off-by: Jan N. Klug <jan.n.klug@rub.de>

### CODE REVIEW

**Comment by J-N-K** [comment]
This is WIP until I can get the tests running again. An issue was reported that should be covered by tests.

**Comment by J-N-K** [comment]
Seems to be fine. No longer WIP from my side.

File: bundles/org.openhab.binding.dmx/src/main/java/org/openhab/binding/dmx/internal/handler/ChaserThingHandler.java [hunk 1]
```diff
@@ -53,7 +52,7 @@
 public class ChaserThingHandler extends DmxThingHandler {
     public static final Set<ThingTypeUID> SUPPORTED_THING_TYPES = Collections.singleton(THING_TYPE_CHASER);
 
-    private static Logger logger = LoggerFactory.getLogger(ChaserThingHandler.class);
+    private Logger logger = LoggerFactory.getLogger(ChaserThingHandler.class);
```
Comment (cweitkamp) [inline bundles/org.openhab.binding.dmx/src/main/java/org/openhab/binding/dmx/internal/handler/ChaserThingHandler.java]: Loggers should be `final`.

Comment (J-N-K) [reply bundles/org.openhab.binding.dmx/src/main/java/org/openhab/binding/dmx/internal/handler/ChaserThingHandler.java]: done

**Pull request merged**